### PR TITLE
Added footer component

### PR DESCRIPTION
**Pull request by adilatilganallocadia**
@alpatilgan as requested in #6 Footer component added to project.

### CODE REVIEW

File: src/app/common/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<p>
+  This will be footer
+
```
Comment (alpatilgan) [inline src/app/common/footer/footer.component.html:3]: Why o you have too many spaces

File: src/app/app.module.ts [hunk 2]
```diff
@@ -3,10 +3,12 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { FooterComponent } from './common/footer/footer.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    FooterComponent
```
Comment (alpatilgan) [inline src/app/app.module.ts:11]: What is a footer component?

File: src/app/common/footer/footer.component.html (hunk 1 above)
Comment (alpatilgan) [reply src/app/common/footer/footer.component.html:3]: Please get rid of all the extra spaces

Comment (adilatilganallocadia) [reply src/app/common/footer/footer.component.html:3]: Good catch removed all of them

File: src/app/app.module.ts (hunk 2 above)
Comment (adilatilganallocadia) [reply src/app/app.module.ts:11]: Footer component will be our footer. Every web site has Footer dow below the page which you can put more information about your product, company and also legal issues policy and licensing.

**Review by alpatilgan — APPROVED** [review]

**Pull request merged**